backend/src/main/java/gov/cdc/usds/simplereport/api/model/Role.java

```diff
@@ -26,16 +26,16 @@ public enum Role {
    * and view historical test results.
    */
   USER(OrganizationRole.USER),
-  /**
-   * This is the organization admin role: if you have this role, then you have the ability to change
-   * your role, so other roles you may have are moot.
-   */
-  ADMIN(OrganizationRole.ADMIN),
   /**
    * This is a specific role for users who are in the test result bulk upload feature pilot and have
    * the ability to submit test results as a CSV
    */
-  TEST_RESULT_UPLOAD_USER(OrganizationRole.TEST_RESULT_UPLOAD_USER);
+  TEST_RESULT_UPLOAD_USER(OrganizationRole.TEST_RESULT_UPLOAD_USER),
+  /**
+   * This is the organization admin role: if you have this role, then you have the ability to change
+   * your role, so other roles you may have are moot.
+   */
+  ADMIN(OrganizationRole.ADMIN);
```

Comment:
So from our original work on this, my impression was that if `ADMIN` came after `TEST_RESULT_UPLOAD_USER` then all admin's would get the csv upload permission. (based on this [note](https://github.com/CDCgov/prime-simplereport/blob/dfe9f57b1bd4ea721721ade782572f27f0d765d0/backend/src/main/java/gov/cdc/usds/simplereport/api/model/Role.java#L12-L17).

But I've tested this locally and that seems to not be the case?  We should double verify in a lower env

Reply:
I think that note is because of [this code](https://github.com/CDCgov/prime-simplereport/blob/3a32786b60ffd0bfa034128ebf38b4688e34b0cb/backend/src/main/java/gov/cdc/usds/simplereport/api/model/Role.java#L65-L70). looks like in the current way we have it, a User with uploader permissions and admin permissions in Okta will return just `TEST_RESULT_UPLOAD_USER` when we call [getRoles()](https://github.com/CDCgov/prime-simplereport/blob/3a32786b60ffd0bfa034128ebf38b4688e34b0cb/backend/src/main/java/gov/cdc/usds/simplereport/api/model/User.java#L49), which I'm guessing is why we're having weird behavior in the Manage Users form when it checks if we're an admin before letting us edit. I think right now any admin with uploader privileges can't edit their users' access in prod.

with this change, a user with both permissions in Okta will appear as an Admin anywhere we use that method so I'd be more concerned if we check for uploader permissions somewhere that they'd fail for someone with both. someone who's only an admin shouldn't be affected, and same for someone who is only an upload user (if that's even a case that exists). kinda want to double-check this tomorrow.

Reply:
I wonder if in the future it's better to have users have only 1 role. so if we do this kind of thing again we create a new role and give it all the permissions of ADMIN plus whatever new ones we want to expose but only assign users to one or the other in Okta. now that I think about it, that might've been more the intention behind permissions vs roles.

Reply:
Looks like we're checking permissions instead of role for the CSV uploader so we're good!